app/poros/trail.rb

```diff
@@ -0,0 +1,45 @@
+class Trail
+  include Coordinatable
+
+  attr_reader :location,
+              :forecast,
+              :trails,
+              :fake_id
+
+  def initialize(destination)
+    @location = destination
+    @latitude = coordinates.latitude
+    @longitude = coordinates.longitude
+    @forecast =  forecast
+    @trails = nearby_trails
```

Comment:
I wonder if any of these should be using `||=` instead so that the method only runs once and then holds onto the return value